docs/source/configuration_file.rst

```diff
@@ -324,11 +324,12 @@ to the tailored architecture.
 
 - ``name``: ``Unet`` (default)
 - ``dropout_rate``: Float (e.g. 0.4).
-- ``batch_norm_momentum``: Float (e.g. 0.1).
-- ``depth``: Strictly positive integer. Number of down-sampling operations. - ``relu`` (optional): Bool.
-  Sets final activation to normalized ReLU (relu between 0 and 1), instead of sigmoid. Only available when `is_2D=True`.
-- ``is_dim``: Indicates dimensionality of model (2D or 3D). If ``is_dim`` is ``False``, then parameters ``length_3D`` and
-  ``stride_3D`` for 3D loader need to be specified (see :ref:`Modified3DUNet <Modified3DUNet>`).
+- ``bn_momentum``: Float (e.g. 0.1).
+- ``depth``: Strictly positive integer. Number of down-sampling operations. 
+- ``final_activation`` (optional): Final activation layer. Options: ``sigmoid`` (default), ``relu`` 
+(normalized ReLU), or ``softmax``.
+- ``is_2d``: Indicates if the model is 2d, if not the model is 3d. If ``is_2d`` is ``False``, then parameters
```

Comment:
how do you feel about changing that for `dim`?  It could then generalize to 1d (optical signal, EEG traces) and 4d (cardiac flow data, DWI data, etc.). If you think it's relevant let's open an issue.

Reply:
Currently, there is no Conv4D PyTorch class, so even if the data is 1d or 2d, we'll treat it as 3d data. For the 1d signals, my understanding is that we would also treat it as 2d (or 3d) data: electrodes and signal / electrode (2d). For now, I'd go simply with 2d or 3d.